source/includes/examples/_find-the-center.html.md

```diff
@@ -0,0 +1,182 @@
+# Find the Center Example
+
+In this example, we'll walk you through the various statements that are part of the Find the Center project, including the simulator file and the Inkling file. Find the Middle was created as a very basic example of Inkling and how to connect to a simulator.
+
+The general premise of Find the Center is that the computer (via Inkling) takes a step (move) of -1, 0, or 1, and gets rewarded for staying in the center (1) for each round. The max is 2 and the min is 0.
```

Comment:
Instead of "computer" what about AI or BRAIN? 

What about starting with the goal of the game and then describing how the player moves... Find the center is a simple game where the AI seeks the average value between two numbers. In this version of the game, the AI begins at a random value of 0, 1, or 2.  The AI then can move to a lower number by outputing -1, a higher number by outputing +1, or staying  on the same number by outputing 0.  The goal of find the center is to remain on the average (the number 1).